sql/src/test/java/ddlog/JooqProviderTest.java

```diff
@@ -129,6 +129,14 @@ public void testSqlOpsNoBindings() {
         assertFalse(goodHostsResultsAfterDelete.contains(test3));
     }
 
+    @Test
+    public void testInsertNull() {
+        // Issue 1036
+        create.insertInto(table("hosts"))
+                .values("n1", 10, null)
```

Comment:
Can you also add test cases for reads, deletes and updates? They have different code paths (that should eventually terminate in toValue() or maybeOption(), but it'd be still good to check.